homeassistant/components/starline/sensor.py

```diff
@@ -30,47 +31,57 @@
         translation_key="battery",
         device_class=SensorDeviceClass.VOLTAGE,
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="balance",
         translation_key="balance",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="ctemp",
         translation_key="interior_temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="etemp",
         translation_key="engine_temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="gsm_lvl",
         translation_key="gsm_signal",
         native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="fuel",
         translation_key="fuel",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="errors",
         translation_key="errors",
+        native_unit_of_measurement="errors",
         entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="mileage",
         translation_key="mileage",
         native_unit_of_measurement=UnitOfLength.KILOMETERS,
         device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.TOTAL,
     ),
```

Comment:
Should this be TOTAL_INCREASING? But only if its strictly increasing

Reply:
Well, unless you unwind the counter - yes :D Changed

Reply:
Lmaooooo

Reply:
It's more like, some APIs are a bit flaky and then you'd have 1-0-1-2-1-3 etc, in which case we should not use total increasing